sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -4,41 +4,73 @@
 
 ```ts
 
+import * as coreClient from '@azure/core-client';
+import * as coreHttps from '@azure/core-rest-pipeline';
 import { OperationOptions } from '@azure/core-client';
 import { PagedAsyncIterableIterator } from '@azure/core-paging';
 import { PipelineOptions } from '@azure/core-rest-pipeline';
 import { TokenCredential } from '@azure/core-auth';
 
+// @public
+export type ArtifactArchitecture = "386" | "amd64" | "arm" | "arm64" | "mips" | "mipsle" | "mips64" | "mips64le" | "ppc64" | "ppc64le" | "riscv64" | "s390x" | "wasm" | string;
+
+// @public
+export interface ArtifactManifestProperties {
+    readonly architecture?: ArtifactArchitecture | null;
```

Comment:
what does `null` vs `undefined` mean? Does it remove the property at the source when doing a patch/update?

 question applies to other nullable properties too

Reply:
These are output only.  It's an error from copy-&-pasting the generated code.

Reply:
Removed `| null`